packages/frontend/src/components/NavBar/index.tsx

```diff
@@ -29,18 +29,31 @@ import SettingsMenu from './SettingsMenu';
 import UserMenu from './UserMenu';
 
 export const NAVBAR_HEIGHT = 50;
-const PREVIEW_BANNER_HEIGHT = 20;
+export const BANNER_HEIGHT = 30;
 
 const PreviewBanner = () => (
-    <Center pos="fixed" w="100%" h={PREVIEW_BANNER_HEIGHT} bg="blue.6">
+    <Center pos="fixed" w="100%" h={BANNER_HEIGHT} bg="blue.6">
         <MantineIcon icon={IconTool} color="white" size="sm" />
-        <Text color="white" fw={500} fz="xs">
+        <Text color="white" fw={500} fz="xs" mx="xxs">
             This is a preview environment. Any changes you make here will not
             affect production.
         </Text>
     </Center>
 );
 
+const DashboardExplorerBanner: FC<{ dashboardName: string }> = ({
+    dashboardName,
+}) => {
+    return (
+        <Center w="100%" h={BANNER_HEIGHT} bg="blue.6">
+            <MantineIcon icon={IconInfoCircle} color="white" size="sm" />
+            <Text color="white" fw={500} fz="xs" mx="xxs">
+                You are creating this chart from within {dashboardName}
+            </Text>
```

Comment:
I think we are missing a button here to cancel the chart creation and go back to the dashboard page.